app/assets/v2/js/pages/process_bounty.js

```diff
@@ -117,6 +117,7 @@ window.onload = function(){
 
                 }
             };
+            // TODO: This call is no longer valid when using Standard Bounties
             bounty.bountydetails.call(issueURL, callback);
```

Comment:
per comment on the main thread, i think we need to build in the 'accept bounty' flow from bounties.network

Reply:
Right -- I agree.  Is there a workflow/UI for doing this now?  Or does it have to be built?

Reply:
https://gitcoin.co/funding/process is the current UI for this